lexicons/app/bsky/unspecced/getTaggedSuggestions.json

```diff
@@ -0,0 +1,43 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.unspecced.getTaggedSuggestions",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Get a list of suggestions (feeds and users) tagged with categories",
+      "parameters": {
+        "type": "params",
+        "properties": {}
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["suggestions"],
+          "properties": {
+            "suggestions": {
+              "type": "array",
+              "items": {
+                "type": "ref",
+                "ref": "#suggestion"
+              }
+            }
+          }
+        }
+      },
+      "errors": [{ "name": "UnknownFeed" }]
```

Comment:
I assume this may just be a detail from `getTimelineSkeleton`.
```suggestion
```

Reply:
lmao rip yup